mstickerlib/src/tg/sticker.rs

```diff
@@ -11,33 +11,22 @@ use std::path::Path;
 #[cfg(feature = "log")]
 use log::info;
 
+///see <https://core.telegram.org/bots/api#photosize>
 #[derive(Clone, Debug, Deserialize, Hash)]
-pub struct Sticker {
-	/// Emoji associated with the sticker.
-	pub emoji: Option<String>,
+pub struct PhotoSize {
```

Comment:
This struct is poorly named. I would expect it to store the size of a photo. However, it stores the file of the "photo" along with its size. It would probably be better to call this `ImageWithSize` or `SizedImage` or similar.

Reply:
I did not how to name. The project does already have many meta-data struct.  So I have simple choose the same name like the telegram api. https://core.telegram.org/bots/api#photosize
Since deserializing is the only usecase for this struct, I am fine with the name.